_includes/footer.html

```diff
@@ -31,7 +31,7 @@ <h6>Knowledge Base</h6>
                     <ul class="menu vertical">
                         <li><a href="{{ site.baseurl }}/support/">Support</a></li>
                         <li><a href="{{ site.baseurl }}/docs/">Documentation</a></li>
-                        <li><a href="https://help.openmicroscopy.org/">Help</a></li>
+                        <li><a href="https://omero-guides.readthedocs.io/">Help</a></li>
```

Comment:
Maybe the link text should be "Guides" now.

Reply:
fixed in https://github.com/ome/www.openmicroscopy.org/pull/401/commits/67afcb0365c3c04285ac81667da80cbf56e2b5ca